tests/test_qpcr_dilution_calculation.py

```diff
@@ -0,0 +1,42 @@
+from itertools import cycle
+
+from scripts.qpcr_dilution_calculation import QPCRDilutionCalculation
+from tests.test_common import TestEPP, FakeEntitiesMaker
+
+
+class TestQPCRDilutionCalculation(TestEPP):
+
+    def setUp(self):
+       self.epp = QPCRDilutionCalculation(self.default_argv)
+
+    def test_qpcr_dilution_calculation(self):
```

Comment:
This test should be transferred in TestGenerateHamiltonInputQPCRDilution

Reply:
Will delete. TestGenerateHamiltonInputQPCRDilution already tests this funcitonality.